ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -2821,6 +2821,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-ipi-ovn-ipsec
+- as: vsphere-ipi-ovn-ipsec-fips-f28-security-profiles-operator
+  cron: 21 12 24 * *
+  steps:
+    cluster_profile: vsphere-2
+    env:
+      FIPS_ENABLED: "true"
+      TEST_FILTERS: ~HyperShiftMGMT
+      TEST_PARALLEL: "3"
+      TEST_SCENARIOS: The Security Profiles Operator
```

Comment:
@kuiwang02 If we add specific jobs for security-profiles-operator, shall we exclude those tests from the default filter that will be executed on the other profiles ?

Reply:
@xiaojiey @liangxia 
Yes. but need to do some changes:

firstly, suggest to change the case code from `[sig-isc] Security_and_Compliance The Security Profiles Operator` to `[sig-isc] Security_and_Compliance The Security_Profiles_Operator`
please note: you could need to cherry pick the change for more release branches if needed.

secondly, you define the test as
```yaml
    env:
      FIPS_ENABLED: "true"
      TEST_FILTERS: ~HyperShiftMGMT
      TEST_PARALLEL: "3"
      TEST_SCENARIOS: Security_Profiles_Operator
```
or
```yaml
    env:
      FIPS_ENABLED: "true"
      TEST_FILTERS: ~HyperShiftMGMT&;Security_Profiles_Operator&
      TEST_PARALLEL: "3"
      TEST_SCENARIOS: Security_and_Compliance

```

lastly, we could change the default value of the step to add `~Security_Profiles_Operator&` for TEST_FILTERS

please finish the first two steps and then merge this PR, and I will change the last steps.

Reply:
@kuiwang02 Do you mean to skip test for all CI jobs because a job for release purpose added? If so, a job with the interval 28 days is far from enough.